mercury/templates/index.html

```diff
@@ -25,7 +29,9 @@
       <br>
       <h1>NYU Motorsports Telemetry</h1>
       <br>
-      <button onclick=window.location="https://nyu-mercury-grafana.herokuapp.com/d/BjN37GJZk/nyu-mercury-poc-prod" class="grafana-btn grafana-btn-primary" style="width: 50%; height: 80px; font-size: larger;">Telemetry Dashboard</button>
+      <button onclick = "newWindow()" class="grafana-btn grafana-btn-primary" style="width: 50%; height: 80px; font-size: larger;"> Telemetry Dashboard</button>
```

Comment:
Since the function isn't reused, I am just wondering why we are creating a function instead of just calling window.open() here.